### PR TITLE
Windmill AI handle multiple lines

### PR DESCRIPTION
**Pull request by hcourdent**

### CODE REVIEW

File: frontend/src/lib/components/apps/svelte-select/lib/tailwind.css [hunk 1]
```diff
@@ -127,4 +127,14 @@
 
 .list-item {
     @apply list-none;
-}
\ No newline at end of file
+}
+
+.textarea-resize-none {
```
Comment (hugocasa) [inline frontend/src/lib/components/apps/svelte-select/lib/tailwind.css]: https://tailwindcss.com/docs/resize
resize-none

File: frontend/src/lib/components/apps/svelte-select/lib/tailwind.css [hunk 2]
```diff
@@ -127,4 +127,14 @@
...
+    resize: none; /* Prevent manual resizing */
+  }
+  
+  .fixed-height-btn {
+    height: 38px; /* Set a fixed height for the button */
```
Comment (hugocasa) [inline frontend/src/lib/components/apps/svelte-select/lib/tailwind.css]: https://tailwindcss.com/docs/height 
h-[38px]

**Review by HugoCasa — COMMENTED** [review]
you should be able to use tailwind classes instead

File: frontend/src/lib/components/apps/svelte-select/lib/tailwind.css (hunk 1 above)
Comment (hcourdent) [reply frontend/src/lib/components/apps/svelte-select/lib/tailwind.css]: Thanks! Fixed for both

**Review by HugoCasa — APPROVED** [review]

**Pull request merged**